.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,74 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
we can think about introducing docker here. 
Thanks to this, you will not need to install packages in this script manually because they will already be in our Docker image. 
Secondly, and more importantly, anyone will be able to download a docker image and run their project without the need to install dependencies.

Please add task to backlog. Task tite: "Introduce docker in project"